File: ago-dic-2023/AlfonsoRamírezCárdenas/Practica1.java
package Algoritmos;

import java.util.Scanner;

public class Practica1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.println("Ingresa dos enteros separados por un espacio:");
        
        // Leer los enteros desde la entrada estándar
        int numero1 = scanner.nextInt();
        int numero2 = scanner.nextInt();
        
        // Calcular la suma
        int suma = numero1 + numero2;
        
        // Imprimir la suma en la salida estándar
        System.out.println("La suma de los enteros es: " + suma);
        
        // Cerrar el scanner
        scanner.close();
    }
}

File: ago-dic-2024/practicas/practica_1/practica1.py
entrada = input("Ingresa dos números enteros separados por un espacio: ")

# Dividimos la entrada en dos números
numero1, numero2 = map(int, entrada.split())

# Calculamos la suma
suma = numero1 + numero2

# Mostramos el resultado
print(f"La suma de {numero1} y {numero2} es: {suma}")

